ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/networking/libp2p/rpc/BlobSidecarsByRangeRequestMessage.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright ConsenSys Software Inc., 2023
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.spec.datastructures.networking.libp2p.rpc;
+
+import tech.pegasys.teku.infrastructure.ssz.containers.Container2;
+import tech.pegasys.teku.infrastructure.ssz.containers.ContainerSchema2;
+import tech.pegasys.teku.infrastructure.ssz.primitive.SszUInt64;
+import tech.pegasys.teku.infrastructure.ssz.schema.SszPrimitiveSchemas;
+import tech.pegasys.teku.infrastructure.ssz.tree.TreeNode;
+import tech.pegasys.teku.infrastructure.unsigned.UInt64;
+
+public class BlobSidecarsByRangeRequestMessage
+    extends Container2<BlobSidecarsByRangeRequestMessage, SszUInt64, SszUInt64>
+    implements RpcRequest {
+
+  public static class BlobSidecarsByRangeRequestMessageSchema
+      extends ContainerSchema2<BlobSidecarsByRangeRequestMessage, SszUInt64, SszUInt64> {
+
+    public BlobSidecarsByRangeRequestMessageSchema() {
+      super(
+          "BlobSidecarsByRangeRequestMessage",
+          namedSchema("startSlot", SszPrimitiveSchemas.UINT64_SCHEMA),
```

Comment:
Let's do it in a snake case, like in [spec](https://github.com/ethereum/consensus-specs/blob/dev/specs/deneb/p2p-interface.md#blobsidecarsbyrange-v1). I've fixed it in other classes to avoid confusion any more: https://github.com/ConsenSys/teku/pull/6863

Reply:
Done